cumulusci/cli/service.py

```diff
@@ -157,7 +157,7 @@ def callback(*args, **kwargs):
             validator_path = service_config.get("validator")
             if validator_path:
                 validator = import_global(validator_path)
-                validator(serv_conf)
+                serv_conf = validator(serv_conf)
```

Comment:
CHECK: Is the return value semantic of other validator methods compatible with this?